src/nsls2api/services/slack_service.py

```diff
@@ -21,7 +27,7 @@ def get_bot_details() -> SlackBot:
     Returns:
         SlackBot: An instance of the SlackBot class containing the bot details.
     """
-    response = app.client.auth_test()
+    response = get_boring_app().client.auth_test()
```

Comment:
🤪

Reply:
Maybe this settles into something more boring, like `get_app()` or `get_unprivileged_app()`?

Reply:
Indeed - probably might go the other way as `get_app()` and `get_privileged_app()`